swagger/swagger.yaml

```
@@ -1078,6 +1078,20 @@ paths:
             security:
                 - oauth2: []
             summary: Delete deployment instance
+    /deployments/public:
+        get:
+            description: Find all public deployments
+            operationId: listPublicDeployments
```

```suggestion
            operationId: findPublicDeployments
```

Or call it List all public deployments